PowerShell/JumpCloud Module/Public/Authentication/Connect-JCOnline.ps1

```diff
@@ -217,25 +177,27 @@ Function Connect-JCOnline ()
                         Write-Host ('JumpCloudOrgName:') -BackgroundColor:($JCColorConfig.BackgroundColor) -ForegroundColor:($JCColorConfig.ForegroundColor_Header)
                         Write-Host ($JCColorConfig.IndentChar) -BackgroundColor:($JCColorConfig.BackgroundColor) -ForegroundColor:($JCColorConfig.ForegroundColor_Indentation) -NoNewline
                         Write-Host ($Auth.JCOrgName) -BackgroundColor:($JCColorConfig.BackgroundColor) -ForegroundColor:($JCColorConfig.ForegroundColor_Body)
+                        # Process Module Notifications:
+                        if (($JCConfig.parallel.MessageCount -le 5) -AND ($JCConfig.parallel.Eligible)) {
```

Comment:
If Parallel.messageCount is less than or equal to 5, notify users about the change to parallel settings